### ISSUE TITLE
Feature Request Pen/Stylus Support

### ISSUE THREAD

**Issue by wiccan2** [Fmstrat/ownnote#241]
Pen/Stylus support in the app would be a great improvement and make this app a good competitor to SNote and the like. Even if it's just an image that gets inserted it would be a big improvement.

**Comment by Fmstrat** [comment]
Can you describe what features of the SPen you would like included? It should work with the SPen keyboard already I would think. I.E. just describe how you would use the SPen in the application. Are you asking for a way to use it like the Samsung Notes app, where it's a drawing?

**Comment by wiccan2** [comment]
Yes exactly, I would love to be able to draw using the pen and have it
embedded into the note. Just a simple vector drawing would do.

It's really frustrating that the SNote app doesn't sync with any of the
services I use and if I could add simple drawings into my ownnote files it
would suite my way of working much better.

I'm not expecting it to do shape recognition or handwriting recognition,
nothing fancy.

The following libraries may be of use:

https://github.com/ayltai/Android-Lib-Pen

http://www.vectoroid.com

I hope this is a feature you can add to the app as it would make for a
richer user experience.

Hope this helps you understand what I meant by this feature.

If you need anymore information don't hesitate to ask.

Thanks

Antony Jordan

**Comment by Fmstrat** [comment]
Yup. Not sure when this would make it in, but it's on the enhancement list.
